scarb-metadata/tests/command.rs

```diff
@@ -1,14 +1,22 @@
 use assert_fs::TempDir;
 use snapbox::cmd::{cargo_bin, Command};
+use std::env;
+use std::path::PathBuf;
 
 use scarb_metadata::MetadataCommand;
 
+fn scarb_bin() -> PathBuf {
+    env::var_os("SCARB_BIN_PATH")
```

Comment:
```suggestion
    env::var_os("SCARB")
```

let's use our standard var name?

Reply:
I have used a different var on purpose, not to confuse with other configuration when running tests locally on user env.

Reply:
Hmm, this was not obvious from me from `SCARB_BIN_PATH` name alone 😕 Your reasoning seems right, so perhaps rename this to something obviously telling that this influences tests?